<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZEMAS</class>
 <widget class="QWidget" name="ZEMAS">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>18</pointsize>
    <bold>true</bold>
    <stylestrategy>PreferAntialias</stylestrategy>
   </font>
  </property>
  <property name="windowTitle">
   <string>App</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-1</y>
     <width>1282</width>
     <height>728</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="minimumSize">
       <size>
        <width>1280</width>
        <height>720</height>
       </size>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <property name="iconSize">
       <size>
        <width>16</width>
        <height>16</height>
       </size>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideMiddle</enum>
      </property>
      <property name="usesScrollButtons">
       <bool>false</bool>
      </property>
      <property name="documentMode">
       <bool>true</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
      <property name="tabBarAutoHide">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="ConfigurationTab">
       <attribute name="title">
        <string>Configuration</string>
       </attribute>
       <widget class="QWidget" name="gridLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>-11</x>
          <y>-1</y>
          <width>1103</width>
          <height>732</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="ConfigurationGrid">
         <item row="0" column="2">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Minimum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="0" colspan="2">
          <widget class="QScrollArea" name="scrollArea">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents_2">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>730</width>
              <height>730</height>
             </rect>
            </property>
            <widget class="QWidget" name="gridLayoutWidget_5">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>0</y>
               <width>781</width>
               <height>701</height>
              </rect>
             </property>
             <layout class="QGridLayout" name="gridLayout_2">
              <item row="4" column="1">
               <widget class="QDoubleSpinBox" name="diffRotSpinBox">
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
                <property name="maximum">
                 <double>360.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.100000000000000</double>
                </property>
               </widget>
              </item>
              <item row="4" column="2">
               <widget class="QDoubleSpinBox" name="imRotSpinBox">
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
                <property name="maximum">
                 <double>360.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.100000000000000</double>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Minimum</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="3" column="3">
               <spacer name="horizontalSpacer_4">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="3" column="2">
               <widget class="QDial" name="imRotDial">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="maximum">
                 <number>359</number>
                </property>
                <property name="wrapping">
                 <bool>true</bool>
                </property>
                <property name="notchTarget">
                 <double>5.000000000000000</double>
                </property>
                <property name="notchesVisible">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="5" column="1">
               <spacer name="verticalSpacer_7">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="2" column="1">
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Image Rotation</string>
                </property>
               </widget>
              </item>
              <item row="1" column="3">
               <widget class="QPushButton" name="browseCIF">
                <property name="text">
                 <string>Browse CIF Structure Files</string>
                </property>
               </widget>
              </item>
              <item row="2" column="2">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Diffraction Rotation</string>
                </property>
               </widget>
              </item>
              <item row="0" column="3">
               <spacer name="verticalSpacer_5">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Minimum</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>10</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="3" column="1">
               <widget class="QDial" name="diffRotDial">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="maximum">
                 <number>359</number>
                </property>
                <property name="wrapping">
                 <bool>true</bool>
                </property>
                <property name="notchTarget">
                 <double>5.000000000000000</double>
                </property>
                <property name="notchesVisible">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="4" column="0">
               <spacer name="verticalSpacer_6">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Minimum</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="TiltingAcquisitionTab">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <attribute name="title">
        <string>Tilting and Acquisition</string>
       </attribute>
       <widget class="QWidget" name="gridLayoutWidget_3">
        <property name="geometry">
         <rect>
          <x>-1</x>
          <y>0</y>
          <width>1281</width>
          <height>641</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <property name="leftMargin">
          <number>5</number>
         </property>
         <property name="rightMargin">
          <number>5</number>
         </property>
         <property name="horizontalSpacing">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QScrollArea" name="scrollArea_2">
           <property name="autoFillBackground">
            <bool>true</bool>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="lineWidth">
            <number>0</number>
           </property>
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>1271</width>
              <height>639</height>
             </rect>
            </property>
            <widget class="QWidget" name="gridLayoutWidget_4">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>670</width>
               <height>581</height>
              </rect>
             </property>
             <layout class="QGridLayout" name="gridLayout_4">
              <property name="leftMargin">
               <number>5</number>
              </property>
              <property name="rightMargin">
               <number>5</number>
              </property>
              <property name="bottomMargin">
               <number>10</number>
              </property>
              <property name="spacing">
               <number>0</number>
              </property>
              <item row="11" column="0">
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Alpha:</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item row="1" column="0" colspan="3">
               <widget class="QComboBox" name="scopeComboBox">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="font">
                 <font>
                  <pointsize>18</pointsize>
                  <bold>true</bold>
                  <underline>false</underline>
                  <stylestrategy>PreferAntialias</stylestrategy>
                 </font>
                </property>
                <property name="autoFillBackground">
                 <bool>true</bool>
                </property>
                <property name="insertPolicy">
                 <enum>QComboBox::InsertAlphabetically</enum>
                </property>
                <property name="frame">
                 <bool>true</bool>
                </property>
                <item>
                 <property name="text">
                  <string>Load from file</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="1" column="5">
               <widget class="QPushButton" name="acquireImage">
                <property name="text">
                 <string>Acquire Image</string>
                </property>
               </widget>
              </item>
              <item row="4" column="2">
               <widget class="QDoubleSpinBox" name="wBox1">
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
                <property name="minimum">
                 <double>-50.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>50.000000000000000</double>
                </property>
               </widget>
              </item>
              <item row="4" column="5">
               <widget class="QPushButton" name="calcZoneAxis">
                <property name="text">
                 <string>Calculate Orientation and Zone Axis</string>
                </property>
               </widget>
              </item>
              <item row="5" column="0">
               <widget class="QDoubleSpinBox" name="uBox2">
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
                <property name="minimum">
                 <double>-50.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>50.000000000000000</double>
                </property>
               </widget>
              </item>
              <item row="5" column="2">
               <widget class="QDoubleSpinBox" name="wBox2">
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
                <property name="minimum">
                 <double>-50.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>50.000000000000000</double>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <layout class="QHBoxLayout" name="horizontalLayout"/>
              </item>
              <item row="5" column="5">
               <widget class="QPushButton" name="moveToZoneAxis">
                <property name="text">
                 <string>Move to Zone Axis</string>
                </property>
               </widget>
              </item>
              <item row="11" column="1">
               <widget class="QDoubleSpinBox" name="alphaSpinBox">
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
                <property name="maximum">
                 <double>360.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.100000000000000</double>
                </property>
               </widget>
              </item>
              <item row="11" column="4">
               <spacer name="horizontalSpacer_5">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Minimum</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>5</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="9" column="1">
               <widget class="QDial" name="alphaDial">
                <property name="maximum">
                 <number>359</number>
                </property>
                <property name="wrapping">
                 <bool>true</bool>
                </property>
                <property name="notchTarget">
                 <double>5.000000000000000</double>
                </property>
                <property name="notchesVisible">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="9" column="3">
               <widget class="QDial" name="betaDial">
                <property name="maximum">
                 <number>359</number>
                </property>
                <property name="wrapping">
                 <bool>true</bool>
                </property>
                <property name="notchTarget">
                 <double>5.000000000000000</double>
                </property>
                <property name="notchesVisible">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="13" column="5">
               <spacer name="verticalSpacer_4">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>5</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="0" column="5">
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Minimum</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>10</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="4" column="0">
               <widget class="QDoubleSpinBox" name="uBox1">
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
                <property name="minimum">
                 <double>-50.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>50.000000000000000</double>
                </property>
               </widget>
              </item>
              <item row="11" column="2">
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Beta:</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item row="1" column="3">
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Minimum</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>10</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="5" column="1">
               <widget class="QDoubleSpinBox" name="vBox2">
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
                <property name="minimum">
                 <double>-50.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>50.000000000000000</double>
                </property>
               </widget>
              </item>
              <item row="10" column="5">
               <spacer name="verticalSpacer_8">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Minimum</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>10</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="8" column="0">
               <spacer name="verticalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Minimum</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="4" column="1">
               <widget class="QDoubleSpinBox" name="vBox1">
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
                <property name="minimum">
                 <double>-50.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>50.000000000000000</double>
                </property>
               </widget>
              </item>
              <item row="13" column="0">
               <spacer name="verticalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Minimum</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>5</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="11" column="5">
               <widget class="QPushButton" name="pushButton">
                <property name="text">
                 <string>Move to Alpha/Beta</string>
                </property>
               </widget>
              </item>
              <item row="11" column="3">
               <widget class="QDoubleSpinBox" name="betaSpinBox">
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
                <property name="maximum">
                 <double>360.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.100000000000000</double>
                </property>
               </widget>
              </item>
              <item row="12" column="2">
               <spacer name="verticalSpacer_9">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
            <widget class="QLabel" name="currentImage">
             <property name="geometry">
              <rect>
               <x>720</x>
               <y>20</y>
               <width>480</width>
               <height>640</height>
              </rect>
             </property>
             <property name="minimumSize">
              <size>
               <width>480</width>
               <height>480</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Tomography</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>4DSTEM</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
